src/pages/home/HomePage.js

```diff
@@ -0,0 +1,74 @@
+import React, {useEffect, useState} from 'react';
+import axios from "axios";
+import './HomePage.css';
+
+import {Link} from "react-router-dom";
+import logo from "../../assets/logo.png";
+import Header from "../../components/header/Header";
+
+function HomePage() {
+    const apiKey = `https://www.reddit.com/hot.json?limit=15`;
+    const [redditPost, setRedditPost] = useState([])
+
+    useEffect(() => {
+        async function fetchData() {
+            try {
+                const result = await axios.get(apiKey);
+                console.log(result.data.data.children);
+                setRedditPost(result.data.data.children)
+            } catch (e) {
+                console.error(e);
+            }
+        }
+
+        fetchData();
+    }, [redditPost])
```

Comment:
Volgens mij moet de dependency array hier leeg zijn, omdat er binnen de useEffect geen dependency ligt op `redditPost`
```suggestion
    }, [])
```